app/services/katello/candlepin/consumer.rb

```diff
@@ -13,7 +13,6 @@ class Consumer
       CP_TYPES = [SYSTEM, HYPERVISOR, CANDLEPIN].freeze
 
       lazy_accessor :entitlements, :initializer => lambda { |_s| Resources::Candlepin::Consumer.entitlements(uuid) }
-      lazy_accessor :events, :initializer => lambda { |_s| Resources::Candlepin::Consumer.events(uuid) }
```

Comment:
Looks like the Owner resource also has an events API and we aren't using it. We can remove that too

```
[vagrant@centos7-devel katello]$ git grep "\.events" app/
app/models/katello/glue/candlepin/owner.rb:        lazy_accessor :events, :initializer => lambda { |_s| Resources::Candlepin::Owner.events(label) }
```